src/main/java/ymfc/recipe/RecommendedRecipe.java

```diff
@@ -0,0 +1,51 @@
+package ymfc.recipe;
+
+import java.util.ArrayList;
+
+/**
+ * Represents the parameters associated with a recipe to recommend to the user.
+ * Contains the {@code Recipe} itself, the percentage of the recipe's ingredients
+ * that matches what the user has, and the missing ingredients the user doesn't have.
+ *
+ * Works in conjunction with {@code SortByPercentMatch} such that an arraylist of
+ * {@code RecommendedRecipe} can be sorted by what is the most accessible to the user
+ * (how easily they can make the recipe without having to obtain additional ingredients).
+ */
+public class RecommendedRecipe{
+
+    private Recipe recipe;
+    private int percentMatch;
+    private ArrayList<String> missingIngredients;
+
+    public RecommendedRecipe(Recipe recipe, int percentMatch, ArrayList<String> missingIngredients) {
+        this.recipe = recipe;
+        this.percentMatch = percentMatch;
+        this.missingIngredients = missingIngredients;
+    }
```

Comment:
Could consider inheriting from Recipe class?

Reply:
For this, my thought process is that the RecommendRecipe class doesn't use any of the methods or attributes of the Recipe class, beyond needing to save one Recipe object, so I didn't think it necessary for them to have an inheritance relationship